src/components/tab-group/tab/bl-tab.css

```diff
@@ -41,6 +42,16 @@
   border-right: none;
 }
 
+:host(:focus-visible) {
+  outline: none;
+}
+
+:host(:focus-visible) .container, .container:focus-visible {
+  outline: 2px solid var(--bl-color-primary);
+  outline-offset: -3px;
```

Comment:
According to our sizing standards, 3px looks like a wrong value. 🤔

Reply:
changed it to 4px (size-3xs).